### PR TITLE
only import rxjs modules that are used

### PR DESCRIPTION
**Pull request by ruddell**
See https://github.com/jhipster/generator-jhipster/pull/6899

### CODE REVIEW

File: src/service/event-manager.service.ts [hunk 1]
```diff
@@ -17,7 +17,10 @@
  limitations under the License.
  */
 import { Injectable } from '@angular/core';
-import { Observable, Observer, Subscription} from 'rxjs/Rx';
+import { Observable } from 'rxjs/Observable';
+import { Observer } from 'rxjs/Observer';
+import { Subscription } from 'rxjs/Subscription';
+import 'rxjs/add/operator/filter';
```
Comment (deepu105) [inline src/service/event-manager.service.ts]: Why is this needed?

Comment (ruddell) [reply src/service/event-manager.service.ts]: filter is used on Line 53 https://github.com/ruddell/ng-jhipster/blob/8c56718f84fa49c1b72f0d71e35888c419230da4/src/service/event-manager.service.ts#L53

Comment (ruddell) [reply src/service/event-manager.service.ts]: The full explanation is in the referenced issue.  By only importing the operators that are used, the import size of rxjs is massively reduced.

Comment (deepu105) [reply src/service/event-manager.service.ts]: Ok thanks
